File: eliminacion_posicion.cs
using System;

class Program
{
    static void Main()
    {
        int[] inputArr = { 11, 21, 31, 41, 51, 61 };
        int position = 3;

        Console.WriteLine("Antes de la eliminación, el array es: ");
        for (int j = 0; j < inputArr.Length; j++)
        {
            Console.Write(inputArr[j] + " ");
            Console.Write("\nDespués de la eliminación, el array es: \n");
            for (int k = 0; k < inputArr.Length; k++)
            {
                Console.Write(inputArr[k] + " ");
            }
        }
    }
}

File: eliminacion_posicion.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<int> inputArr = {11, 21, 31, 41, 51, 61};
    int position = 3;

    std::cout << "Antes de la eliminación, el array es: \n";
    for (size_t j = 0; j < inputArr.size(); ++j) {
        std::cout << inputArr[j] << " ";
        std::cout << "\nDespués de la eliminación, el array es: \n";
        for (size_t j = 0; j < inputArr.size(); ++j) {
            std::cout << inputArr[j] << " ";
        }
    }
    return 0;
}